spec/models/billing_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+RSpec.describe Billing, type: :model do
+  context 'with associations' do
+    it 'belongs to user' do
+      billing = create(:billing, user: create(:user))
+      expect(billing).to respond_to :user
+    end
+  end
+
+  context 'with validation' do
```

Comment:
It would be better to use `let` in the top
```
context 'when invalid zip' do
  let(:billing) { build(:billing, zip: '49000')

  it 'is invalid with no integer zip' do
      expect(billing).not_to be_valid
    end
```